ui/components/app/add-network/add-network.test.js

```diff
@@ -6,6 +6,7 @@ import mockState from '../../../../test/data/mock-state.json';
 import AddNetwork from './add-network';
 
 jest.mock('../../../selectors', () => ({
+  ...jest.requireActual('../../../selectors'),
```

Comment:
There must be a reason why the extension team did not want to require the actual selectors and mock only some of them. I see there are no other changes in the file so probably you added it because it is now needed down the tree. 
To follow the same patter I would not require the whole file but add only the mock (or mocks) we are missing for the tests.

Unless an extension team member is OK with requiring the whole selectors file (my rationale here is to follow the existing patterns)

Reply:
yep, the issue here is that this test and other tests that mock the selectors object fail due to the new `getIsNativeTokenBuyable` selector that uses `createSelector` because `getCurrentChainId` is not defined. See below.   Requiring the actual selector is a decent way to make sure that all the other selectors are available.

```
● Test suite failed to run

    Selector creators expect all input-selectors to be functions, instead received the following types: [undefined, function]

      46 | export const getBuyableChains = (state: any) => state.ramps.buyableChains;
      47 |
    > 48 | export const getIsNativeTokenBuyable = createSelector(
         |                                                      ^
      49 |   [getCurrentChainId, getBuyableChains],
      50 |   (currentChainId, buyableChains) => {
      51 |     return buyableChains.some(
```